packages/fgtclb/typo3-category-types/Classes/Loader/CategoryTypeLoader.php

```diff
@@ -58,8 +58,51 @@ public function loadUncached(): array
                 }
                 if (array_key_exists('types', $configArray) && is_array($configArray['types'])) {
                     foreach ($configArray['types'] as $categoryType) {
+                        // Check if the identifier and group of the category type to ensure it is unambiguous
```

Comment:
I guess we need to add a generic array structure validation ... the handling within this loop is quite incomplete, even with the now added additions.

And more importingly, not covered by tests.

Reply:
Added code comment/todo. Resolved, leaving open to be regonized.